typescript/nomad-deploy/src/core/index.ts

```diff
@@ -452,10 +452,10 @@ export async function enrollRemote(local: CoreDeploy, remote: CoreDeploy) {
 /**
  * Transfers governorship to the governing chain's GovernanceRouter.
  *
- * @param gov - The governor chain deploy instance
  * @param non - The non-governor chain deploy instance
+ * @param gov - The governor chain deploy instance
  */
-export async function transferGovernorship(gov: CoreDeploy, non: CoreDeploy) {
+export async function transferGovernorship(non: CoreDeploy, gov: CoreDeploy) {
```

Comment:
why?

Reply:
because transferring from right => left made more intuitive sense to me than the other way

Reply:
i'll undo

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/3/commits/30a0b0d23e7171f6c01fc89835659545bca08415